generators/liquibase/templates/src/main/resources/config/liquibase/master.xml.ejs

```diff
@@ -74,6 +80,7 @@
     <property name="blobType" value="blob" dbms="neo4j"/>
     <property name="uuidType" value="string" dbms="neo4j"/>
     <property name="datetimeType" value="date" dbms="neo4j"/>
+    <property name="timeType" value="localtime" dbms="neo4j"/>
```

Comment:
? is this the right type?

Reply:
Thanks for the docu link. Googled it, but didn't find it.

Changed to `date` accordingly.